File: src/data/snippets/if-statements/snippet.cpp
#include <iostream>

int main() {
    // Identify age group
    int age = 22;

    if (age <= 12) {
        std::cout << "You're a kid!" << std::endl;

    } else if (age <= 18) {
        std::cout << "You're a teen!" << std::endl;

    } else if (age <= 60) {
        if (age <= 25) {
            std::cout << "You're a young adult!" << std::endl;

        } else {
            std::cout << "You're an adult!" << std::endl;
        }
        
    } else {
        std::cout << "You're old!" << std::endl;
    }

    return 0;
}

File: src/data/snippets/if-statements/snippet.py
# Identify age group
age = 22

if age <= 12:
    print("You're a kid!")
    
elif age <= 18:
    print("You're a teen!")

elif age <= 60:
    if age <= 25:
        print("You're a young adult!")
        
    else:
        print("You're an adult!")

else:
    print("You're old!")
